tools/deploy.sh

```diff
@@ -210,6 +210,18 @@ fi
 
 if [ "${DEPLOY_IRONIC}" == "true" ]; then
     pushd "${SCRIPTDIR}"
+    if [[ "${DEPLOY_KEEPALIVED}" == "true" ]]
+    then
+      IRONIC_BMO_CONFIGMAP="${SCRIPTDIR}/ironic-deployment/keepalived/ironic_bmo_configmap.env"
+    else
+      IRONIC_BMO_CONFIGMAP="${SCRIPTDIR}/ironic-deployment/default/ironic_bmo_configmap.env"
+    fi
+    if grep "INSPECTOR_REVERSE_PROXY_SETUP" "${IRONIC_BMO_CONFIGMAP}" > /dev/null
+    then
+      sed "s/\(INSPECTOR_REVERSE_PROXY_SETUP\).*/\1=${DEPLOY_TLS}/" -i "${IRONIC_BMO_CONFIGMAP}"
+    else
+      echo "INSPECTOR_REVERSE_PROXY_SETUP=${DEPLOY_TLS}" >> "${IRONIC_BMO_CONFIGMAP}"
```

Comment:
you should add something to revert those changes. at the end of this script we try to restore the state of the repo. Another solution would be to copy the whole thing to a temporary folder and delete it afterwards.

Reply:
Fixed! Thanks